Lib/asyncio/futures.py

```diff
@@ -417,10 +423,56 @@ def wrap_future(future, *, loop=None):
     return new_future
 
 
+def future_add_to_awaited_by(fut, waiter, /):
+    """Record that `fut` is awaited on by `waiter`."""
+    # For the sake of keeping the implementation minimal and assuming
+    # that 99.9% of asyncio users use the built-in Futures and Tasks
```

Comment:
```suggestion
    # that most of asyncio users use the built-in Futures and Tasks
```

let's not use numbers without data

Reply:
🫡